### ISSUE TITLE
users: parsing of dates for lastlog in user details page is wrong

### ISSUE THREAD

**Issue by KKoukiou** [cockpit-project/cockpit#15452]
If the inspected user has an entry in last log and it's not currently connected it always displays invalid date.

![Screen Shot 2021-03-02 at 16 45 27](https://user-images.githubusercontent.com/14921356/109673977-d1bb4300-7b76-11eb-8b84-08d8d5cdda38.png)

**Comment by martinpitt** [comment]
I can't reproduce this on my local machine. I have only two local user accounts, "mpitt" (for installing my system) and "root"):
```
❱❱❱ lastlog | egrep 'mpitt|root'
root                                       **Never logged in**
mpitt            pts/1                     Sat Mar  6 17:47:04 +0100 2021
```
... and that's exactly what I see in Cockpit:
![image](https://user-images.githubusercontent.com/200109/112144787-e6b94e00-8bd9-11eb-836d-1909b2fc710a.png)

![image](https://user-images.githubusercontent.com/200109/112144807-ee78f280-8bd9-11eb-9a83-d807e8244c7f.png)

However, your screenshot sounds like our test VMs, so I tried that as well with `bots/vm-run -s cockpit.socket fedora-34`. The Accounts page shows the correct values for all accounts: "admin" (Logged in), "builder" (March 23, 2021 1:13 PM, supposedly image-prepare time), "fedora" (Never) and "root" (Never).

I now created a "jussi" user on the cockpit Accounts page. It's shown as "last login: never". I ssh in, back out, and it moves to the correct time.

@KKoukiou , do you still remember the conditions/OS where you saw this?

**Comment by KKoukiou** [comment]
@martinpitt the reproducer is pretty straightforward, no need for test VMs.

* Create a 'test' user while being connected to mpitt
* Log out from your 'mpitt' account and log in with 'test' user
* Last login should be 'Logged in' at this point for your 'test' user
* Log out from the 'test' user and log in with the 'mpitt' account
* Inspect the 'test' user details page, 'invalid date' should be at the 'last login' field at this point

**Comment by martinpitt** [comment]
Fixed in #15575 -- this was fun!
